app-web/src/logIn.tsx

```diff
@@ -24,20 +36,20 @@ const ButtonWrapper = styled.div({
 });
 
 interface Props {
-  register: any;
-  errors: any;
-  handleSubmit: any;
-  login: any;
+  register: UseFormRegister<User>;
+  errors: FieldErrorsImpl<DeepRequired<User>>;
+  handleSubmit: UseFormHandleSubmit<User>;
+  login: UseMutateFunction<any, unknown, User, unknown>;
 }
 
 export default function LogIn({ register, errors, handleSubmit, login }: Props) {
   return (
     <FormWrapper
       style={{ padding: '1rem 0' }}
-      onSubmit={handleSubmit(async (data: any) => {
-        await login({
-          email: data.email,
-          password: data.password,
+      onSubmit={handleSubmit(({ email, password }: User) => {
+        login({
+          email,
+          password,
         });
```

Comment:
```suggestion
      onSubmit={handleSubmit(login)};
```

이렇게 해도 되나요? `handleSubmit`의 타입이 어떤지 확인해보고, 파라미터가 하나밖에 없다면 위처럼 해도 될 것 같아요

Reply:
login이 mutate 부분인데...!
인자로 email 과 password가 리덕스에 저장되지 않아
```js
onSubmit={() => handleSubmit(login(email, password))} 
```
위와 같은 방식으로 해야할 것 같은데...!
이런 방식으로 해도 괜찮을까요!?

Reply:
아 그러넹. 원래 코드가 맞네요